<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaoHelpTemplate</class>
 <widget class="QMainWindow" name="TaoHelpTemplate">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>372</x>
      <y>520</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>我知道了</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkBoxDisplay">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>520</y>
      <width>131</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开始时不再显示</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVisitWeb">
    <property name="geometry">
     <rect>
      <x>685</x>
      <y>520</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>访问官网</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txtBrowserHelpContent">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>500</height>
     </rect>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
